src/gameflow.h

```diff
@@ -43,7 +43,7 @@ class CGameflow
 
     uint8_t             getCurrentGameID();
     uint8_t             getCurrentLevelID();
-    const char*         getCurrentLevelName();
+    const char*         getCurrentLevelName(); ///@TODO: delete unused function!
```

Comment:
I don't recommend deleting this, getCurrentLevelName returns the current level's name. Will be useful for the far future when level end statistics etc will be shown.